app/service/news_api_service.rb

```diff
@@ -0,0 +1,16 @@
+class NewsApiService 
+  def self.search_articles(keyword)
+    response = conn.get("/v2/everything??") do |req|
+      req.params["q"] = "#{keyword}"
+      req.params["pageSize"] = 2
+    end
+    
+    data = JSON.parse(response.body, symbolize_names: true)
```

Comment:
If you do not intend on using the "data" variable elsewhere in class You do not need to set the JSON.parse to a variable as the variable is not used elsewhere in the _Service_. If you are going to use the variable, remember to make it an instance variable (@data) or a class method (@@data)... if so desired.

The parsed json will returned as it is the last functionality in the method.

Reply:
thanks! I changed this to remove the interpolation

Reply:
Great. 
I do still see that the parsing the json is set to a variable. Is there a reason for setting it to a variable?  
If not, it can be removed.

Reply:
I just changed this. We can add it back if need be later.